docs/usage/ipv6.md

```diff
@@ -114,71 +102,58 @@ spec:
+With the next node roll-out, nodes will get IPv6 addresses and an IPv6 prefix for pods. When all nodes have IPv4 and IPv6 pod ranges, the status will be changed to `DualStackNodesReady`.
+The next reconcile will change all the remaining components to DualStack. Once the migration has finished the constraint will be removed with the next reconcile.
 
 ---
 
 ## Load Balancer Service Configuration
 
-When creating LoadBalancer services in a DualStack cluster, you must include specific annotations to enable proper reconciliation by the `ingress-gce` component for IPv6 support.
+To create a DualStack LoadBalancer the `spec.ipFamilies` and `spec.ipFamilyPolicy` field needs to be specified in the kubernetes service.
+An example configuration is shown below:
 
-### Required Annotation for IPv6 LoadBalancers
-Add the following annotation to the service:
-```yaml
-cloud.google.com/l4-rbs: enabled
 ```
-
-### Internal Load Balancer Considerations
-- Internal IPv6 LoadBalancers are **not supported**.
-- For internal IPv4 LoadBalancers, you can use:
-  - `"networking.gke.io/load-balancer-type=Internal"`
-  - `"cloud.google.com/load-balancer-type=internal"` (deprecated).
-  They are created by cloud-controller-manger and get an an IPv4 address from the internal subnet.
-
-### Example Configuration
-Here is an example of a DualStack LoadBalancer service configuration:
-
-```yaml
 apiVersion: v1
 kind: Service
 metadata:
+  name: nginx
+  namespace: default
   annotations:
     cloud.google.com/l4-rbs: enabled
-  name: webapp2
-  namespace: default
 spec:
-  ipFamilyPolicy: RequireDualStack
   ipFamilies:
   - IPv4
   - IPv6
+  ipFamilyPolicy: PreferDualStack
   ports:
-  - port: 80
-    targetPort: 80
+  - port: 12345
     protocol: TCP
+    targetPort: 80
   selector:
-    run: webapp2
+    run: nginx
   type: LoadBalancer
 ```
 
+The required `cloud.google.com/l4-rbs: enabled` annotation for ingress gce is added automatically via webhook for services of `type: LoadBalancer`.
+
 ### Explanation
 - **`cloud.google.com/l4-rbs: enabled`**: Ensures that `ingress-gce` properly reconciles the LoadBalancer service with IPv6 support.
 - **`ipFamilyPolicy` and `ipFamilies`**: Specify the DualStack configuration.
-- **Internal LoadBalancer**: Use the specific annotations for internal IPv4-only LoadBalancers.
----
+
+### Internal Load Balancer
+- Internal IPv6 LoadBalancers are **not supported**.
+- To create internal IPv4 LoadBalancers, you can set one of the the following annotations:
+  - `"networking.gke.io/load-balancer-type=Internal"`
+  - `"cloud.google.com/load-balancer-type=internal"` (deprecated).
+  Internal load balancers are created by cloud-controller-manger and get an an IPv4 address from the internal subnet.
+
 
 ## Key Benefits of DualStack Support
```

Comment:
Let's remove this section and use the important parts of the content in the introduction/overview sections.